script/tool/lib/src/version_check_command.dart

```diff
@@ -123,47 +123,49 @@ class VersionCheckCommand extends PackageLoopingCommand {
             '(e.g., PR description or commit message).\n\n'
             'If supplied, this is used to allow overrides to some version '
             'checks.');
+    argParser.addOption(_prLabelsArg,
+        help: 'A comma-separated list of labels associated with this PR, '
+            'if applicable.\n\n'
+            'If supplied, this may be to allow overrides to some version '
+            'checks.');
     argParser.addFlag(_checkForMissingChanges,
         help: 'Validates that changes to packages include CHANGELOG and '
             'version changes unless they meet an established exemption.\n\n'
-            'If used with --$_changeDescriptionFile, this is should only be '
-            'used in pre-submit CI checks, to  prevent the possibility of '
-            'post-submit breakage if an override justification is not '
-            'transferred into the commit message.',
+            'If used with --$_prLabelsArg, this is should only be '
+            'used in pre-submit CI checks, to  prevent post-submit breakage '
+            'when labels are no longer applicable.',
         hide: true);
     argParser.addFlag(_ignorePlatformInterfaceBreaks,
         help: 'Bypasses the check that platform interfaces do not contain '
             'breaking changes.\n\n'
             'This is only intended for use in post-submit CI checks, to '
-            'prevent the possibility of post-submit breakage if a change '
-            'description justification is not transferred into the commit '
-            'message. Pre-submit checks should always use '
-            '--$_changeDescriptionFile instead.',
+            'prevent post-submit breakage when overriding the check with '
+            'labels. Pre-submit checks should always use '
+            '--$_prLabelsArg instead.',
         hide: true);
   }
 
   static const String _againstPubFlag = 'against-pub';
   static const String _changeDescriptionFile = 'change-description-file';
+  static const String _prLabelsArg = 'pr-labels';
   static const String _checkForMissingChanges = 'check-for-missing-changes';
   static const String _ignorePlatformInterfaceBreaks =
       'ignore-platform-interface-breaks';
 
-  /// The string that must be in [_changeDescriptionFile] to allow a breaking
+  /// The label that must be on a PR to allow a breaking
   /// change to a platform interface.
-  static const String _breakingChangeJustificationMarker =
-      '## Breaking change justification';
+  static const String _breakingChangeOverrideLabel =
+      'override: allow breaking change';
```

Comment:
Even if people can change the name of the label inadvertently in the Github UI, the result is that the check CANNOT be overridden, so the failure mode is safety, good stuff!